### PR TITLE
[FIX] fix the python script for building resnet (#6526)

### PR DESCRIPTION
**Pull request by intheworld**
Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

### CODE REVIEW

**Comment by intheworld** [comment]
@jroesch @ehsanmok please review this change.

File: rust/tvm/examples/resnet/build.rs [hunk 1]
```diff
@@ -23,6 +23,7 @@ use std::{path::Path, process::Command};
 fn main() -> Result<()> {
     let output = Command::new("python3")
         .arg(concat!(env!("CARGO_MANIFEST_DIR"), "/src/build_resnet.py"))
+        //.arg(&format!("--pretrained"))
```
Comment (mbaret) [inline rust/tvm/examples/resnet/build.rs:26]: Left in by accident?

Comment (intheworld) [reply rust/tvm/examples/resnet/build.rs:26]: No, I intent to add this. Someone who wants to use a pre-trained model could just uncomment this line. I think this is more convenient.

Comment (tqchen) [reply rust/tvm/examples/resnet/build.rs:26]: Thanks @intheworld, I also agree that we should keep the pretrained flag as it is

Comment (intheworld) [reply rust/tvm/examples/resnet/build.rs:26]: Ok, I will remove this line.

**Review by mbaret — APPROVED** [review]

**Review by tqchen — APPROVED** [review]

**Pull request merged**